akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/StashManagement.scala

```diff
@@ -55,3 +56,24 @@ private[akka] trait EventsourcedStashManagement[C, E, S] {
   }
 
 }
+
+/**
+ * INTERNAL API
+ * Main reason for introduction of this trait is stash buffer reference management
+ * in order to survive restart of internal behavior
+ */
+@InternalApi private[akka] trait StashReferenceManagement {
```

Comment:
Relocated to here, because it seems unnecessary to be in it's own, given they are both Stash*Management and small.